clients/src/main/java/org/apache/kafka/common/requests/FindCoordinatorResponse.java

```diff
@@ -45,12 +43,12 @@
     private final Errors error;
     private final Node node;
 
-    public GroupCoordinatorResponse(Errors error, Node node) {
+    public FindCoordinatorResponse(Errors error, Node node) {
```

Comment:
I filed https://issues.apache.org/jira/browse/KAFKA-5053 and will add it to the EOS epic so we don't lose track of this work item.